demo/server/src/modules/wfm-web-api/DataService.ts

```diff
@@ -0,0 +1,35 @@
+import { Db } from 'mongodb';
+
+/**
+ * Service for performing data operations on mongodb database
+ */
+export class DataService {
+
+  /**
+   * @param db - mongodb driver connection
+   * @param collectionName - name of the collection stored in mongodb
+   */
+  constructor(readonly db: Db, readonly collectionName: string) {
+  }
+
+  public list() {
+    return this.db.collection(this.collectionName).find({});
+  }
```

Comment:
I think this class, or an interface definition of it should be async, not all storage libs can fit a synchronous api.